src/docs/reference/modules/Sms/README.md

```diff
@@ -0,0 +1,37 @@
+# SMS (`OrchardCore.Sms`)
+
+This module provides the infrastructure necessary to send messages using `SMS` service.
+
+## SMS Settings
+
+Enabling the `OrchardCore.Sms` will add a new settings under `Configurations` >> `Settings` >> `SMS`. You can utilize these settings to set up the default SMS provider configuration. The following are the providers that are readily accessible.
```

Comment:
Enabling the `OrchardCore.Sms` will add a new settings under
=>
Enabling the `OrchardCore.Sms` feature will add a new settings page under

Reply:
Done.